### PR TITLE
Restyled Login page, restyled for mobile.

### PR DESCRIPTION
**Pull request by ziyielenachen**
1. Made Login Page prettier and works for mobile
2. Fixed the header of each page to fit mobile screen size
3. Fixed font size and row height in react table (DataTable.css) to fit mobile screen size (no more "..." for column names)
Screen Shot 2019-12-08 at 6 34 21 PM
Screen Shot 2019-12-08 at 7 41 05 PM
Screen Shot 2019-12-08 at 7 41 37 PM
Screen Shot 2019-12-08 at 4 55 12 PM
Screen Shot 2019-12-08 at 5 07 32 PM
Screen Shot 2019-12-08 at 5 10 31 PM

### CODE REVIEW

**Comment by ziyielenachen** [comment]
If you go to login page and the ethical logo is not turning or you get any error related to "styled", install this in your ethical-internal-website folder: npm install --save styled-components

File: src/components/Login.css [hunk 1]
```diff
@@ -1,24 +1,73 @@
...
+  font-family: 'Solway', serif;
+}
+.pic {
+  width: 9.2vh;
+  height: auto;
 }
+.pic2 {
+  width: 10vh;
+  height: auto;
+}
```
Comment (abaruah117) [inline src/components/Login.css]: Add new line

File: src/components/Login.js [hunk 2]
```diff
@@ -55,34 +85,37 @@ class Login extends Component {
...
+              onChange={this.handleChange}
+            />
+            <Button primary
+              className='button'
+              onClick={() => this.buttonClick()}
+              id="button"
+              type="submit"
+              value="Login"
+            >
+              {" "}
```
Comment (abaruah117) [inline src/components/Login.js]: Remove these empty spaces

**Review by abaruah117 — CHANGES_REQUESTED** [review]
Small changes, also put images in diff folder please! Instead of in components

**Comment by ziyielenachen** [comment]
All changes made.

**Review by trev4ev — DISMISSED** [review]
lgtm

**Review by abaruah117 — DISMISSED** [review]
lgtm

**Comment by abaruah117** [comment]
@ziyielenachen wait can you move the images folder outside of components? so it'd be src/images

**Pull request merged**